### PR TITLE
Add DracFun to addon list

### PR DESCRIPTION
**Pull request by PhoenixCodingStuff**
~~Also when will my application be reviewed.~~

### CODE REVIEW

**Comment by svr333** [comment]
Your application for what?

While the requirements don't say it has to be open-source I think it is enforced? Though I'm not sure so @Sefiraat what do you think?

And personally I find that
> I will probably open source in the future because lot of code is not complete and under comments

this should not be a reason to not open-source, sometimes code is just quick and messy but if it works it works, who cares what it looks like. Though that is just my opinion

**Comment by Sefiraat** [comment]
Open source is a requirement for being on the builds repo but it's not one for the wiki as far as I know, and In my opinion I have no issues with it being closed here. That being said I do, without reservation, recommend you make it open source for many reasons. - regardless of your thoughts on its completeness and code quality. You will not be judged on that, but you may well be judged on the closed source nature, but that's your choice I guess.

File: pages/Addons.md [hunk 1]
```diff
@@ -89,6 +89,7 @@ VillagerTrade | ybw0014 | <details><summary>expand</summary>This addon allows se
...
 AdvancedTech | PranavVerma-droid | <details><summary>expand</summary>AdvancedTech is an immersive add-on that brings exciting new things like handheld diggers and new solar panels!</details> | [Readme](https://github.com/PranavVerma-droid/AdvancedTech#readme) | [Github](https://github.com/PranavVerma-droid/AdvancedTech) | [Builds](https://thebusybiscuit.github.io/builds/PranavVerma-droid/AdvancedTech/dev/)
 Quaptics | Idra | <details><summary>expand</summary>Quaptics adds Magic Quantum Lasers™, large multiblocks, an excessive amount of eye candy, and an overcomplicated electric-circuit inspired energy system. What more do you want?</details> | [Readme](https://github.com/LordIdra/Quaptics#readme) | [Github](https://github.com/LordIdra/Quaptics) | [Builds](https://thebusybiscuit.github.io/builds/LordIdra/Quaptics/master/)
+DracFun | Phoenix | <details><summary>expand</summary>DracFun adds new powerful tools, machines, and items which are based upon the plugin's main resource - Draconium.</details> | [Readme](https://modrinth.com/plugin/dracfun) | N/A | [Modrinth](https://modrinth.com/plugin/dracfun/versions)
```
Comment (TheBusyBiscuit) [inline pages/Addons.md:92]: Don't call it a readme, if it's not a readme file.
```suggestion
DracFun | Phoenix | <details><summary>expand</summary>DracFun adds new powerful tools, machines, and items which are based upon the plugin's main resource - Draconium.</details> | [Modrinth](https://modrinth.com/plugin/dracfun) | N/A | [Modrinth](https://modrinth.com/plugin/dracfun/versions)
```

**Review by TheBusyBiscuit — CHANGES_REQUESTED** [review]

**Comment by TheBusyBiscuit** [comment]
> I will probably open source in the future because lot of code is not complete and under comments👍
> ~Also it is unoptimized asf. [...]
> The requirements did not say it needs to be open source so I will update GitHub link in the future with PR 

Yep, that's fine 👍🏻
The projects do not need to be open-source to be listed here but open-source is undisputedly superior ;)

**Review by TheBusyBiscuit — APPROVED** [review]

**Review by Phoenix-Starlight — APPROVED** [review]
Hopefully the pull bot doesn't force push this, no comments aside. Welcome to the addon list.

**Pull request merged**